Formula/luajit.rb

```diff
@@ -64,6 +64,8 @@ def install
               "INSTALL_LMOD=#{HOMEBREW_PREFIX}/share/lua/${abiver}"
       s.gsub! "INSTALL_CMOD=${prefix}/${multilib}/lua/${abiver}",
               "INSTALL_CMOD=#{HOMEBREW_PREFIX}/${multilib}/lua/${abiver}"
+      s.gsub! "Libs: ",
+              "Libs: -pagezero_size 10000 -image_base 100000000 "
```

Comment:
Do you want me to remove the last space from both lines? Or am I misunderstanding?